config/serverless.yml

```diff
@@ -229,3 +229,8 @@ monitoring.ui.enabled: false
 xpack.securitySolution.enableUiSettingsValidations: true
 data.enableUiSettingsValidations: true
 discover.enableUiSettingsValidations: true
+
+## Data Usage in stack management
+xpack.dataUsage.enabled: true
```

Comment:
is this config serving any purpose now we have the FF?

Reply:
Yes, we enable the plugin for Serverless only. Non serverless users shouldn't have the ability to enable it with FF